upgrade-2.x.sh

```diff
@@ -101,7 +101,7 @@ function remove_containers() {
     docker stop $(docker ps -a -q) > /dev/null 2>&1 || true
     log "Removing all containers..."
     # shellcheck disable=SC2046
-    docker rm -f $(docker ps -a -q) > /dev/null 2>&1 || true
+    docker rm $(docker ps -a -q) > /dev/null 2>&1 || true
```

Comment:
Would it be possible to just drop this patch and the one that added it?

Reply:
@willnewton I don't quite understand your comment here.

Reply:
@agherzan there were two commits before, one added the "-f" function, and another removed it (thus the line was unchanged in the end). Thus the comment was to drop those two commits altogether instead of having a patch and a revert in a WIP PR. So you are replying to an outdated state.